types/licences.d.ts

```diff
@@ -140,8 +140,12 @@ export interface LduService {
 
 export interface LicenceSearchService {
   findForId: (username: string, id: string) => Promise<number | undefined>
+}
+
+export interface ReportsService {
```

Comment:
If you've defined ReportService as typescript you shouldn't need to define an interface for it - these are just to help typescript work with js files

Reply:
ah, think maybe it's because you've defined the service as a function that returns an object..

If you defined it as a class you'd get this for free. 

To avoid having to redeclare all the functions you could also us the `ReturnType` 
like how we do for dataAccess in any typescript repo:
```ts
const dataAccess = () => ...
export type DataAccess = ReturnType<typeof dataAccess>
```